python/pacemaker/_cts/clustermanager.py

```diff
@@ -720,13 +720,8 @@ def find_partitions(self):
         self.debug("Found partitions: %r" % ccm_partitions)
         return ccm_partitions
 
-    def HasQuorum(self, node_list):
-        # If we are auditing a partition, then one side will
-        #   have quorum and the other not.
-        # So the caller needs to tell us which we are checking
-        # If no value for node_list is specified... assume all nodes
-        if not node_list:
-            node_list = self.Env["nodes"]
+    def HasQuorum(self):
+        node_list = self.Env["nodes"]
```

Comment:
FYI this isn't a bug because existing callers all expect the cluster to not be partitioned

Reply:
I'll just drop this patch, in case we ever end up wanting to test a partitioned cluster.  There are several other functions that take an argument we're not really using that I left alone for the future.  Might as well add this one to that.